src/ApiTrait.php

```diff
@@ -9,7 +9,7 @@
 
 trait ApiTrait
 {
-    private Extractor $extractor;
+    protected Extractor $extractor;
```

Comment:
Because https://github.com/oscarotero/Embed/blob/master/src/Adapters/Facebook/OEmbed.php#L17 and https://github.com/oscarotero/Embed/blob/master/src/Adapters/Instagram/OEmbed.php#L15 are accessing the property. 
We can also use a getter here if you like.